support-frontend/assets/components/socialShare/socialShare.jsx

```diff
@@ -10,45 +10,51 @@ import SvgFacebook from 'components/svgs/facebook';
 import SvgTwitter from 'components/svgs/twitter';
 import SvgLinkedin from 'components/svgs/linkedin';
 import SvgEmail from 'components/svgs/email';
-
+import { trackComponentClick } from 'helpers/tracking/ophanComponentEventTracking';
 
 // ---- Types ----- //
 
-type Platform = 'facebook' | 'twitter' | 'linkedin' | 'email';
+type SharePlatform = 'facebook' | 'twitter' | 'linkedin' | 'email';
 
-type PropTypes = {| name: Platform |};
+type PropTypes = {| name: SharePlatform |};
 
 type SocialMedia = {
   link: string,
   svg: Node,
   a11yHint: string,
+  windowFeatures: string
 };
 
+const SocialWindowFeatures = 'menubar=no, toolbar=no, resizable=yes, scrollbars=yes, width=500, height=400';
 
 // ----- Setup ----- //
 
 const socialMedia: {
-  [Platform]: SocialMedia,
+  [SharePlatform]: SocialMedia,
 } = {
   facebook: {
-    link: 'https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fsupport.theguardian.com%2F%3FINTCMP%3Dsocial&t=',
+    link: 'https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fsupport.theguardian.com%2Fcontribute',
```

Comment:
oh, one more thing on the tracking for these links. I would use this to generate them: http://reader-revenue-lynx.s3-website-eu-west-1.amazonaws.com/

in the second dropdown (source) you can choose EMAIL or SOCIAL and put `contributions-share-facebook`, `contributions-share-twitter`, etc in the Component ID field

Reply:
it seems that facebook does support the urls that this tool generates however twitter strips the entire link out of the tweet when it has this parameter, linkedin strips out the value of the query parameter, and gmail unencodes the json (which probably isn't a technical problem but doesn't look very trustworthy to click)

Reply:
Yep you're right! Those URL-encoded JSON blobs are not good for this use case. Let's leave out the tracking for the purposes of this PR and address in a future PR.

What we can do is create some friendly-URL redirects for these, e.g. https://support.theguardian.com/contribute/twitter-share, which redirect to a page with all the tracking info. We did something similar for the Australian campaign: https://support.theguardian.com/au/contribute/thefrontline

Unfortunately it's not the easiest thing to do right now - for Australia we did it in Fastly which was a bit of a pain (as @tomrf1 will tell you). We should probably make a little tool on the admin console that can set up redirects.

Of course we could always use https://bitly.com/ or similar to shorten the links but people might be less inclined to click on links which don't have the guardian domain.

Reply:
and as per our chat conversation - INTCMP is fine for now